README.md

```diff
@@ -4,3 +4,20 @@ For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/
 This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
 
 1. Once you have cloned the repo, please add your own copy of .env.development and .env.test in the root folder of the project in order to link correctly the databases. Use the following PGDATABASE=data_base_name and PGDATABASE=data_base_name_test respectively.
+
+2. Setup DB:
+   psql -f ./db/setup.sql
+
+3. Seed tables:
+   node ./db/seeds/run-seed.js
+
+Dependencies to install:
+
+0. Making sure npm has been initiated: npm init -y
```

Comment:
npm has already been initiated by you - you can tell this because package.json exists so the users don't need to do this!